pkg/bitcoin/electrum/electrum_integration_test.go

```diff
@@ -0,0 +1,343 @@
+//go:build integration
+// +build integration
+
+package electrum
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"math"
+	"testing"
+	"time"
+
+	"github.com/go-test/deep"
+	"golang.org/x/exp/slices"
+
+	"github.com/keep-network/keep-core/pkg/bitcoin"
+)
+
+// TODO: Include integration test in the CI.
+// To run the tests execute `go test -v -tags=integration ./...`
```

Comment:
When I tried to run this command, I got:
```
pkg/bitcoin/electrum/electrum_integration_test.go:14:2: no required module provides package github.com/go-test/deep; to add it:
        go get github.com/go-test/deep
```

After running  `go get github.com/go-test/deep`, the package got installed and `go.mod` and `go.sum` got updated.

We should probably add this package to `go.mod` and `go.sum` in this PR as well.

Reply:
79a8cdb39aa96a7911ea05c0f9380a1f98243df0